RFC-004-revising-licenses-page.md

```diff
@@ -0,0 +1,118 @@
+# RFC004 - Revising website licenses terms/page
+
+## Proposed by
+
+Ryan Macklin ([@macklin](https://thegooddocs.slack.com/team/U01DYRWG43X))
+Initially submitted on 2021-07-13
+
+## Current status
+
+- [ ] Draft
+- [x] Under discussion (until 2021-07-27, assuming two weeks post initial submission)
+- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+## Proposal overview
+
+I believe we should rethink the licensing terms for website content, as listed on [thegooddocsproject.dev/licenses/](https://thegooddocsproject.dev/licenses/). As they're currently written, they provision everything as [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/) (a.ka. "CC BY 4.0), including all text, images, HTML code, and anything else.
+
+We may not always have the rights to relicense what we put on our page, and not every contributor is comfortable with having their info and likeness released as publicly usable.
```

Comment:
1) There a large difference between use and relicense. Most OS projects have free-licensed code integrated with original code, which is a different paradigm than human content.

2) For me, none of those options solve the issue. i'm especially against A, because that should be someone's decision make for themselves, not because of the licensing options on a project they work on.

Reply:
2. Asking contributors to contribute under a pen name or anonymously is not a great solution. Many people contribute to blogs to raise their professional profiles. Many people who care about being able to control their images are minoritized in tech. This request puts them at a professional disadvantage.

I would vote for prioritizing inclusiveness for blog contributors over making life easier for hypothetical future maintainers.

Reply:
Between the tools that Bryan and I have worked out for the blog, and being able to have an internal document that makes the licensing text clear for our purposes as for which to use, I also see this as making life easy for us in the future as maintainers.

Reply:
I acknowledge and agree with the arguments that we shouldn't compromise user needs (especially the option A). I'll step back from this comment.